docs/releases/5.0.txt

```diff
@@ -37,6 +37,13 @@ compatible with Django 5.0.
 What's new in Django 5.0
 ========================
 
+Facet filters in the admin
+--------------------------
+
+Facet counts are now shown for applied filters in admin change-list UI. By
+default, facet counts can be toggled via a ``_facets`` query parameter, but
```

Comment:
Just one query: do we want to say, "can be toggled in the UI" now, rather than mentioning the query parameter? (Here and in docs.)

Reply:
Yes true, it probably is less confusing also - I've tried to update


There is a picture at the top here which I would like to update, I don't suppose this is someones test app?
![image](https://user-images.githubusercontent.com/42296566/220895184-dc7dec6f-37e4-468b-ae5e-785fc94d97ab.png)

https://docs.djangoproject.com/en/4.1/ref/contrib/admin/filters/


I saw there is this ticket: https://code.djangoproject.com/ticket/34043 which would be generally useful and I could also use for updating documentation screenshots. Might be a good one for a Django sprint maybe

Reply:
I have a pretty elaborate mechanism for automated screenshots for my own project, I wonder if Django might benefit from something similar?

Definitely not as a blocker for this change though!

https://simonwillison.net/2022/Oct/14/automating-screenshots/